plugins/inspect/unmanaged_files_inspector.rb

```diff
@@ -273,9 +304,13 @@ def inspect(system, description, options = nil)
     unmanaged_links = {}
     remote_dirs = mount_points.remote
     special_dirs = mount_points.special
-    ignore_list.each do |ignore|
-      remote_dirs.delete_if { |e| e.start_with?(File.join("/", ignore, "/")) }
-    end
+
+    filter_locator = "/unmanaged_files/files/name"
+    filter_dirs = ElementFilter.new(filter_locator)
```

Comment:
Maybe we could actually leave out the filter locator here (according to our new terminology this should be a path anyway). If the `ElementFilter` is used directly as in this inspector, the path is not relevant. Would it just work to allow a constructor without arguments?

This makes me think a bit, if the path actually should be part of the `ElementFilter` at all, as it's not used in the class itself.